templates/lxd/index.html

```diff
@@ -0,0 +1,350 @@
+{% extends "lxd/base_lxd.html" %}
+
+{% block title %}Run system containers with LXD{% endblock %}
+
+{% block meta_description %}Low-touch virtual infrastructure with LXD at any scale. Built on top of Linux containers, LXD offers a unified user experience in managing system containers, virtual machines and clusters. Easily configurable, flexible and secure way to run your workloads.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <h1>Run system containers with LXD</h1>
+      <p class="p-heading--4">Fast, dense, and secure container and VM management at any scale </p>
+      <p> LXD provides a unified user experience for managing system containers and virtual machines. For more demanding workloads, LXD can be set up in a cluster environment to run containers, VMs, or a combination of the two on a set of machines. LXD has direct hardware access, minimising overhead and matching the density and efficiency of containers.</p>
+      <p>
+        <a class="p-button--positive" href="https://linuxcontainers.org/lxd/try-it/">Try LXD Online</a>
```

Comment:
I get a 'Your browser couldn't reach the demo server' message when trying to access the demo

Reply:
I will double check if the link is correct with the PdM. Thanks Pete!

Reply:
Yes this is the correct link. You should be able to accept the t&cs and create a demo